### PR TITLE
fix invalid html

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #14206.

### CODE REVIEW

File: modules/mod_whosonline/tmpl/default.php [hunk 1]
```diff
@@ -17,10 +17,10 @@
 <?php endif; ?>
 
 <?php if (($showmode > 0) && count($names)) : ?>
-	<ul  class="whosonline<?php echo $moduleclass_sfx; ?>" >
 	<?php if ($params->get('filter_groups')) : ?>
 		<p><?php echo JText::_('MOD_WHOSONLINE_SAME_GROUP_MESSAGE'); ?></p>
 	<?php endif; ?>
+	<ul  class="whosonline<?php echo $moduleclass_sfx; ?>" >
```
Comment (C-Lodder) [inline modules/mod_whosonline/tmpl/default.php]: double space before the class: `<ul  class=` >> `<ul class=`

Comment (C-Lodder) [inline modules/mod_whosonline/tmpl/default.php]: also can remove the space at the end of the element closing tag

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on e91dcbcc60b42a5f0771bf241cb7a3eb39bfff7a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14454">issues.joomla.org/tracker/joomla-cms/14454</a>.</sub>

File: modules/mod_whosonline/tmpl/default.php (hunk 1 above)
Comment (brianteeman) [inline modules/mod_whosonline/tmpl/default.php]: fat fingers. - fixed

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 8eea0e18e5c0970d46592c0205623a362418adc1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14454">issues.joomla.org/tracker/joomla-cms/14454</a>.</sub>

**Pull request merged**